lib/Dialect/LWE/IR/LWETypes.h

```diff
@@ -3,8 +3,31 @@
 
 #include "lib/Dialect/LWE/IR/LWEAttributes.h"
 #include "lib/Dialect/LWE/IR/LWEDialect.h"
+#include "llvm/include/llvm/ADT/TypeSwitch.h"  // from @llvm-project
 
 #define GET_TYPEDEF_CLASSES
 #include "lib/Dialect/LWE/IR/LWETypes.h.inc"
 
+namespace mlir {
+namespace heir {
+namespace lwe {
+
+template <typename Op>
+void lweGetAsmResultNames(Op op, ::mlir::OpAsmSetValueNameFn setNameFn) {
+  for (auto result : op->getResults()) {
+    llvm::TypeSwitch<Type>(result.getType())
```

Comment:
I checked some other examples of `GetAsmResultNames` and apparently "not doing anything" is the correct thing when you don't want to handle an op? It'd be nice to add a comment that the lack of a default case isn't an oversight but intentional.